src/DeprecatedCallableInfo.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Twig;
+
+/**
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+final class DeprecatedCallableInfo
+{
+    private string $type;
+    private string $name;
+
+    public function __construct(
+        private string $package,
+        private string $version,
+        private ?string $altPackage = null,
+        private ?string $altName = null,
+        private ?string $altVersion = null,
```

Comment:
Shouldn't we put `altName` before `altPackage` to keep the package name and package version together ?

Reply:
Good idea, done.